ash/src/vk/definitions.rs

```diff
@@ -1467,24 +1477,24 @@ impl ::std::default::Default for PhysicalDeviceMemoryProperties {
 }
 impl PhysicalDeviceMemoryProperties {
     #[inline]
-    pub fn memory_type_count(mut self, memory_type_count: u32) -> Self {
-        self.memory_type_count = memory_type_count;
+    pub fn memory_types(mut self, memory_types: &'_ [MemoryType]) -> Self {
```

Comment:
I've modified `vk.xml` on this struct as an example, but I believe we can merge this without any generator output changes while making sure the upstream CI doesn't start failing when they make and merge this change :ok_hand:

Reply:
SGTM.

Reply:
Approving conditional on this diff being rolled back.

Reply:
CI would reject those diffs anyway :tada: